src/d3a/models/strategy/external_strategy.py

```diff
@@ -171,7 +171,7 @@ def _offer(self, payload):
             assert set(arguments.keys()) == {'price', 'energy'}
             arguments['seller'] = self.area.name
             arguments['seller_origin'] = self.area.name
-        except Exception as e:
+        except Exception:
```

Comment:
How about instead of ignoring the exception to add a log that displays the error ? Maybe also the `arguments` would be useful to be printed here.

Reply:
yeah, flake complained here. I wanted to be consistent with the `_bid` method. But ok, i will add it to the end of the error log (in both places).